.gitignore

```diff
@@ -428,18 +420,8 @@ DerivedData/
 **/xcshareddata/WorkspaceSettings.xcsettings
 
 /libs
+
 # End of https://www.gitignore.io/api/xcode
 
 # Don't accidently commit the diabdat.mpq
 *.mpq
-
-# Static libs, generated files and directories
```

Comment:
Putting a .gitignore with `*` in the build folder should do, might even commit that.